openfisca_us/tests/policy/baseline/usda/snap/integration/bplc_ny.yaml

```
@@ -0,0 +1,23 @@
+- name: Family of 1, no member(s) with disabilities, income of 0k/year (Standard Test)
+  period: 2021
+  absolute_error_margin: 0.01
+  input:
+    people:
+      person_1:
+        age: 30
+    spm_units:
+      spm_unit:
+        snap_gross_income: 0 * 12
+        members: [person_1]
+    households:
+      household:
+        state_code_str: NY
+        region_str: NY_NYC
+  output:
+    snap_net_income: 0 
```

Could you also move this below the `snap_sua`?
```suggestion
    snap_net_income: 0  # "Monthly Net SNAP Income"
```